src/renderer/component/publishForm/view.jsx

```diff
@@ -555,7 +564,12 @@ class PublishForm extends React.PureComponent<Props> {
           </section>
 
           <div className="card__actions">
-            <Submit label={submitLabel} disabled={formDisabled || !formValid || publishing} />
+            <Submit
+              label={submitLabel}
+              disabled={
+                formDisabled || !formValid || uploadThumbnailStatus === STATUSES.IN_PROGRESS
```

Comment:
This design forces the user to wait for the upload to finish before clicking Publish and it may not be clear why they're waiting.

It would be preferable if Publish could be clicked regardless of the upload status and instead the app waited for the remote upload to finish before making the `publish` API call.

(Fixing this may be tricky, and if it's complicated probably shouldn't be considered blocking.)